### ISSUE TITLE
[cloud_firestore] XCode Build Error due to undeclared Identifier in GRPC

### ISSUE THREAD

**Issue by danielkap** [FirebaseExtended/flutterfire#1533]
Issue has already been described here #1237 
Opening another issue as requested since the original one has been closed.

**Describe the bug**
Running XCode Build after adding the cloud_firestore dependendy causes the following Error.
Happens with old apps as well as with freshly created ones.
```
test_v3 flutter run 
Launching lib/main.dart on iPhone 11 Pro Max in debug mode...
Running pod install...                                              6,5s
Running Xcode build...                                                  
                                                   
Xcode build done.                                            5,6s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/*****/Projects/Testing/test_v3/ios/Pods/BoringSSL-GRPC/ssl/t1_lib.cc:3224:9: error: use of undeclared identifier 'SSL_get_secure_renegotiation_support'; did you mean 'GRPC_SHADOW_SSL_get_secure_renegotiation_support'?
          !(SSL_get_secure_renegotiation_support(ssl) &&
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            GRPC_SHADOW_SSL_get_secure_renegotiation_support
    In file included from /Users/*****/Projects/Testing/test_v3/ios/Pods/BoringSSL-GRPC/ssl/t1_lib.cc:109:
    /Users/*****/Projects/Testing/test_v3/ios/Pods/BoringSSL-GRPC/include/openssl/ssl.h:4328:5: note: expanded from macro 'SSL_get_secure_renegotiation_support'
        SSL_get_secure_renegotiation_support
        ^
    /Users/*****/Projects/Testing/test_v3/ios/Pods/BoringSSL-GRPC/include/openssl/ssl.h:1523:20: note: 'GRPC_SHADOW_SSL_get_secure_renegotiation_support' declared here
    OPENSSL_EXPORT int SSL_get_secure_renegotiation_support(const SSL *ssl);
                       ^
    In file included from <built-in>:1:
    /Users/*****/Projects/Testing/test_v3/ios/Pods/Target Support Files/BoringSSL-GRPC/BoringSSL-GRPC-prefix.pch:232:46: note: expanded from macro 'SSL_get_secure_renegotiation_support'
    #define SSL_get_secure_renegotiation_support GRPC_SHADOW_SSL_get_secure_renegotiation_support
                                                 ^
    1 error generated.
    note: Using new build system
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro Max.

```
**To Reproduce**
Create a new Project via `flutter create`
`flutter run`
-> Works like a charm
Add `cloud_firestore: 0.12.10+2`
`flutter run`
-> Build fails with the mentioned error

**Additional context**
The Fix mentioned here solved it for me 
https://www.gitmemory.com/issue/flutter/flutter/32202/491577147

Flutter Doctor output:
```
flutter doctor -v
[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.15.1 19B88, locale en-DE)
    • Flutter version 1.9.1+hotfix.6 at /Users/***/Applications/Flutter/flutter
    • Framework revision 68587a0916 (vor 3 Monaten), 2019-09-13 19:46:58 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/***/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • Java binary at: /Users/***/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/191.5977832/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.2.1, Build version 11B500
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Users/***/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/191.5977832/Android Studio.app/Contents
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] Android Studio
    • Android Studio at /Users/***/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-1/192.5863777/Android Studio 3.6 Preview.app/Contents
    • Flutter plugin version 37.0.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] VS Code (version 1.40.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (2 available)
    • Android SDK built for x86 • emulator-5554                        • android-x86 • Android 9 (API 28) (emulator)
    • iPhone 11 Pro Max         • E1D6074F-5685-4BF9-B331-C9BFD4A68EBE • ios         • com.apple.CoreSimulator.SimRuntime.iOS-13-2 (simulator)
```

Recently updated to Catalina as well as XCode 11 so the issue might be related to that

**Comment by jrmarkham** [comment]
I have the same issue after upgrading  **cloud_first store to 0.12.11**
Only a problem on iOS devices; android works fine after updates. 

I tried rebuilding iOS version from scratch and the Pods. I tried the fix above but I didn't have those definitions in my header. Note: this project worked fine earlier this week. This happened without code changes beyond the plugin updates.

Thanks,
JM

Errors: 
`Launching lib/main.dart on iPhone 6 Plus in debug mode...
Running Xcode build...
Xcode build done.                                           254.2s
Failed to build iOS app
Error output from Xcode build:
↳
    2019-12-05 11:19:07.353 xcodebuild[50165:1296836] [MT] iPhoneConnect: 📱<DVTiOSDevice (0x7fe398cd5570), JM iPhone X, iPhone, 13.1.3 (17A878), fb23caaec4bfc8e2ad73ab81cb5717a370687766> == Underlying device preparation errors ==
    2019-12-05 11:19:07.353 xcodebuild[50165:1296836] [MT] iPhoneConnect: 📱<DVTiOSDevice (0x7fe398cd5570), JM iPhone X, iPhone, 13.1.3 (17A878), fb23caaec4bfc8e2ad73ab81cb5717a370687766> -- Failed _shouldMakeReadyForDevelopment check even though device is not locked by passcode.
    Domain: com.apple.platform.iphoneos
    Code: 5
    Failure Reason: allowsSecureServices: 1. isConnected: 0. Platform: <DVTPlatform:0x7fe398c02220:'com.apple.platform.iphoneos':<DVTFilePath:0x7fe398c01f70:'/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform'>>. DTDKDeviceIdentifierIsIDID: 0
    --
    2019-12-05 11:19:07.353 xcodebuild[50165:1296836] [MT] iPhoneConnect: 📱<DVTiOSDevice (0x7fe398cd5570), JM iPhone X, iPhone, 13.1.3 (17A878), fb23caaec4bfc8e2ad73ab81cb5717a370687766> == END: Underlying device preparation errors ==
    ** BUILD FAILED **


Xcode's output:
↳
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.4.0/ios/Classes/RateMyAppPlugin.m:15:30: warning: 'requestReview' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    In module 'StoreKit' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.4.0/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:22:1: note: 'requestReview' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    + (void)requestReview API_AVAILABLE(ios(10.3), macos(10.14)) __TVOS_PROHIBITED;
    ^
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.4.0/ios/Classes/RateMyAppPlugin.m:15:30: note: enclose 'requestReview' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.4.0/ios/Classes/RateMyAppPlugin.m:15:6: warning: 'SKStoreReviewController' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    In module 'StoreKit' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.4.0/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:14:85: note: 'SKStoreReviewController' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    SK_EXTERN_CLASS API_AVAILABLE(ios(10.3), macos(10.14)) __TVOS_PROHIBITED @interface SKStoreReviewController : NSObject
                                                                                        ^
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.4.0/ios/Classes/RateMyAppPlugin.m:15:6: note: enclose 'SKStoreReviewController' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    2 warnings generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Write.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    1 warning generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/firestore/local/Target.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    1 warning generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/rpc/Status.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Any.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Any.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Any.pbobjc.h>
    1 warning generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Query.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Wrappers.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Wrappers.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Wrappers.pbobjc.h>
    1 warning generated.
    error: the following command failed with exit code 0 but produced no further output
    CompileC /Users/johnmarkham/Library/Developer/Xcode/DerivedData/Runner-dfcwcnfqiqmhvzfoiecbonbhxdeo/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FirebaseFirestore.build/Objects-normal/x86_64/Mutation.pbobjc.o /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/firestore/local/Mutation.pbobjc.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/firestore/local/MaybeDocument.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    1 warning generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Firestore.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Empty.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Empty.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Empty.pbobjc.h>
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Firestore.pbobjc.m:34:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    2 warnings generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Document.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Struct.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Struct.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Struct.pbobjc.h>
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Document.pbobjc.m:34:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    2 warnings generated.
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1/Common.pbobjc.m:33:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    1 warning generated.
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.12.11/ios/Classes/CloudFirestorePlugin.m:764:16: warning: 'timestampsInSnapshotsEnabled' is deprecated [-Wdeprecated-declarations]
          settings.timestampsInSnapshotsEnabled = (bool)call.arguments[@"timestampsInSnapshotsEnabled"];
                   ^
    In module 'FirebaseFirestore' imported from /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Headers/Public/Firebase/Firebase.h:31:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/FirebaseFirestore/Firestore/Source/Public/FIRFirestoreSettings.h:69:20: note: 'timestampsInSnapshotsEnabled' has been explicitly marked deprecated here
        __attribute__((deprecated));
                       ^
    1 warning and 4 errors generated.
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRewardedVideoAdWrapper.m:67:66: warning: implicit conversion loses integer precision: 'FLTRewardedVideoAdStatus' to 'int' [-Wshorten-64-to-32]
          (NSString *)rewardedStatusToString[[NSNumber numberWithInt:_rewardedStatus]];
                                             ~                       ^~~~~~~~~~~~~~~
    1 warning generated.
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.m:71:13: warning: 'testDevices' is deprecated: Use GADMobileAds.sharedInstance.requestConfiguration.testDeviceIdentifiers. [-Wdeprecated-declarations]
        request.testDevices = testDevices;
                ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:88:71: note: 'testDevices' has been explicitly marked deprecated here
    @property(nonatomic, copy, nullable) NSArray<NSString *> *testDevices GAD_DEPRECATED_MSG_ATTRIBUTE(
                                                                          ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.m:91:15: warning: 'birthday' is deprecated [-Wdeprecated-declarations]
          request.birthday =
                  ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:95:55: note: 'birthday' has been explicitly marked deprecated here
    @property(nonatomic, copy, nullable) NSDate *birthday GAD_DEPRECATED_ATTRIBUTE;
                                                          ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'
    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))
                                                    ^
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.m:106:19: warning: 'gender' is deprecated [-Wdeprecated-declarations]
              request.gender = genderValue;
                      ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:92:47: note: 'gender' has been explicitly marked deprecated here
    @property(nonatomic, assign) GADGender gender GAD_DEPRECATED_ATTRIBUTE;
                                                  ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'
    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))
                                                    ^
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.m:116:14: warning: 'tagForChildDirectedTreatment:' is deprecated:  use [GADMobileAds.sharedInstance.requestConfiguration tagForChildDirectedTreatment] [-Wdeprecated-declarations]
        [request tagForChildDirectedTreatment:childDirected.boolValue];
                 ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:126:5: note: 'tagForChildDirectedTreatment:' has been explicitly marked deprecated here
        GAD_DEPRECATED_MSG_ATTRIBUTE(
        ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTRequestFactory.h:5:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    4 warnings generated.
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTMobileAd.m:40:17: warning: 'configureWithApplicationID:' is deprecated: Use [GADMobileAds.sharedInstance startWithCompletionHandler:] [-Wdeprecated-declarations]
      [GADMobileAds configureWithApplicationID:appId];
                    ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTMobileAd.h:6:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADMobileAds.h:74:5: note: 'configureWithApplicationID:' has been explicitly marked deprecated here
        GAD_DEPRECATED_MSG_ATTRIBUTE("Use [GADMobileAds.sharedInstance startWithCompletionHandler:]");
        ^
    In module 'GoogleMobileAds' imported from /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTMobileAd.h:6:
    /Users/johnmarkham/Work/Flutter/Projects/dailydeath/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /usr/local/lib/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.0+9/ios/Classes/FLTMobileAd.m:98:79: warning: implicit conversion loses integer precision: 'FLTMobileAdStatus' to 'int' [-Wshorten-64-to-32]
      NSString *statusString = (NSString *)statusToString[[NSNumber numberWithInt:_status]];
                                                          ~                       ^~~~~~~
    2 warnings generated.
    note: Using new build system
    note: Planning build
    note: Constructing build description
    warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'in_app_purchase' from project 'Pods')
    warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'in_app_purchase' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 6 Plus.`


YAML
`name: dailydeath
description: Daily Memento Mori
version: 1.0.11+11

environment:
  sdk: ">=2.2.2 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^0.1.3
  firebase_core: ^0.4.2+1
  firebase_auth: ^0.15.1
  cloud_firestore: ^0.12.11
  firebase_admob: ^0.9.0+9
  firebase_messaging: ^6.0.2
  shared_preferences: ^0.5.4+8
  sqflite: ^1.1.7+3
  path_provider: ^1.4.5
  intl: ^0.16.0
  rate_my_app: ^0.4.0
  in_app_purchase: ^0.2.2+3
  connectivity: ^0.4.5+7
  dio: ^3.0.7
  equatable: ^1.0.1
  flutter_bloc: ^2.1.1
  device_info: ^0.4.1+3

dev_dependencies:
  flutter_test:
    sdk: flutter

flutter:
  uses-material-design: true
  assets:
    - assets/json_files/en/core.json
    - assets/json_files/en/daily.json
    - assets/images/logo.png
  fonts:
    - family: Roboto
      fonts:
        - asset: assets/fonts/Roboto-Regular.ttf
    - family: Daily
      fonts:
        - asset: assets/fonts/Daily.ttf
    - family: Distress
      fonts:
        - asset: assets/fonts/distres2.ttf`


DOCTOR: 
`[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.15.1 19B88, locale
    en-US)
    • Flutter version 1.9.1+hotfix.6 at /usr/local/lib/flutter
    • Framework revision 68587a0916 (3 months ago), 2019-09-13 19:46:58 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/johnmarkham/Library/Android/sdk/
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = /Users/johnmarkham/Library/Android/sdk/
    • Java binary at:
      /Applications/JetBrains/apps/AndroidStudio/ch-1/191.5977832/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.2.1, Build version 11B500
    • CocoaPods version 1.8.3

[✓] Android Studio (version 3.5)
    • Android Studio at
      /Applications/JetBrains/apps/AndroidStudio/ch-1/191.5977832/Android
      Studio.app/Contents
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] Android Studio (version 3.5)
    • Android Studio at
      /Applications/JetBrains/apps/AndroidStudio/ch-1/191.5977832/Android
      Studio.app/Contents
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[!] IntelliJ IDEA Ultimate Edition (version 2019.3)
    • IntelliJ at /Users/johnmarkham/Applications/JetBrains Toolbox/IntelliJ
      IDEA Ultimate.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] Connected device (3 available)
    • Android SDK built for x86 • emulator-5554                        •
      android-x86 • Android 10 (API 29) (emulator)
    • iPhone 11 Pro Max         • 0A4F72A2-E8CD-4044-9DBC-C7475D5430D7 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-13-2 (simulator)
    • iPhone 6 Plus             • 3F964A81-152E-439B-812D-93E911E4489A • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-10-3 (simulator)`

**Comment by jrmarkham** [comment]
I got my projects (so far 2 for 2) working with a pods update. The issue for me was the cloud_firestore plugin.

so:

> cd ios; pod update Firebase/Firestore;
> build

**Comment by danielkap** [comment]
Is there any news on this? 
Pod update did not solve it for me. 
Also tried reinstalling cocoapods and flutter-SDK as well as XCode. 
I'm rather new to MacOS so I'm kinda running out of ideas about what might cause this and I'd be glad for any hints.

Attached is the output of flutter doctor -v 
[consoleoutput.txt](https://github.com/FirebaseExtended/flutterfire/files/4049343/consoleoutput.txt)

**Comment by Salakar** [comment]
Hey all, as part of our on-going work for #2582, this has been resolved in our Firebase Firestore rework (#2913) - which has now been merged into master. We'll look at publishing some prereleases in the next few days. Thank you
